### ISSUE TITLE
enable fixed_event in these default feeds to avoid exponential correlations table growth

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#5544]
While experimenting with correlations enabled we detected huge growth in the db `correlations` table and discovered that these 5 feeds had many common attributes being ingested each day into  a new event. This caused correlations to expand quickly. We've set `fixed_event=1` which helped tremendously.

Does it make sense to set `fixed_event = 1` in the MISP release?

```
MariaDB [misp]> select name,url,fixed_event from feeds where url like '%osint.bambenekconsulting.com%';
+-----------------------------------------------------------------------+----------------------------------------------------------------------+-------------+
| name                                                                  | url                                                                  | fixed_event |
+-----------------------------------------------------------------------+----------------------------------------------------------------------+-------------+
| IPs from High-Confidence DGA-Based C&Cs Actively Resolving            | http://osint.bambenekconsulting.com/feeds/c2-ipmasterlist-high.txt   |           0 |
| Domains from High-Confidence DGA-based C&C Domains Actively Resolving | http://osint.bambenekconsulting.com/feeds/c2-dommasterlist-high.txt  |           0 |
| All current domains belonging to known malicious DGAs                 | http://osint.bambenekconsulting.com/feeds/dga-feed-high.csv          |           0 |
| IPs from High-Confidence DGA-Based C&Cs Actively Resolving            | https://osint.bambenekconsulting.com/feeds/c2-ipmasterlist-high.txt  |           0 |
| Domains from High-Confidence DGA-based C&C Domains Actively Resolving | https://osint.bambenekconsulting.com/feeds/c2-dommasterlist-high.txt |           0 |
+-----------------------------------------------------------------------+----------------------------------------------------------------------+-------------+
```

**Comment by github-germ** [comment]
1. Might there be other feeds that deserve this also?
1. Is there a way to `disable_correlations` for feeds?

**Comment by Rafiot** [comment]
I changed the feeds you noted to fixed events by default, and we will review the other ones. A good practice is that you look at the feeds before enabling them blindly.

There is no way to disable correlation for a feed, but you can disable correlation of an event after creation.

**Comment by github-germ** [comment]
Thanks! Here's another one...

* `abuse.ch SSL IPB`

Btw, I think adding the ability to enabled/disable correlations for default feeds would be useful. See https://github.com/MISP/MISP/issues/5568

**Comment by Rafiot** [comment]
That was fixed here: https://github.com/MISP/MISP/commit/9b2d1d2117f998da7f1c9bf5cb686dc0303e2c3b

**Comment by github-germ** [comment]
> That was fixed here: [9b2d1d2](https://github.com/MISP/MISP/commit/9b2d1d2117f998da7f1c9bf5cb686dc0303e2c3b)

Great!! :)
